d2renderers/textmeasure/textmeasure.go

```diff
@@ -76,6 +76,8 @@ type Ruler struct {
 
 	atlases map[d2fonts.Font]*atlas
 
+	ttfs map[d2fonts.Font]*truetype.Font
```

Comment:
maybe a comment the tifs = true type font something 😛

Reply:
the value spells it out